content/source/docs/cloud/registry/index.html.md

```diff
@@ -7,7 +7,7 @@ page_title: "Private Module Registry - Terraform Cloud and Terraform Enterprise"
 
 > **Hands-on:** Try the [Share Modules in the Private Module Registry](https://learn.hashicorp.com/tutorials/terraform/module-private-registry?in=terraform/modules&utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorial on HashiCorp Learn.
 
-Terraform Cloud's private module registry helps you share [Terraform modules](/docs/modules/index.html) across your organization. It includes support for module versioning, a searchable and filterable list of available modules, and a configuration designer to help you build new workspaces faster.
+Terraform Cloud's private module registry helps you share [Terraform modules](/docs/language/modules/develop/index.html) across your organization. It includes support for module versioning, a searchable and filterable list of available modules, and a configuration designer to help you build new workspaces faster.
```

Comment:
```suggestion
Terraform Cloud's private module registry helps you share [Terraform modules](/docs/language/modules/index.html) across your organization. It includes support for module versioning, a searchable and filterable list of available modules, and a configuration designer to help you build new workspaces faster.
```

Reply:
Oh hey, excellent catch! That link should never have pointed to /docs/modules in the first place.